base/applications/mscutils/eventvwr/lang/zh-TW.rc

```diff
@@ -123,13 +123,16 @@ BEGIN
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "記錄檔(&F)：", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "大小：", IDC_STATIC, 7, 57, 60, 8
+// The following 4 IDC_STATIC shall have an accelerator in ReactOS, although MS doesn't have accels for them.
+// Dear Translators: First fill out ALL accels that MS brings as well in this dialog for your language,
+// and only then as a final step use some remaining unused letters for those 4 controls!
+    LTEXT "大小：(&S)", IDC_STATIC, 7, 57, 60, 8
     EDITTEXT IDC_SIZE_LABEL, 67, 57, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "建立日期：", IDC_STATIC, 7, 69, 60, 8
+    LTEXT "建立日期：(&C)", IDC_STATIC, 7, 69, 60, 8
     EDITTEXT IDC_CREATED_LABEL, 67, 69, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "修改日期：", IDC_STATIC, 7, 81, 60, 8
+    LTEXT "修改日期：(&E)", IDC_STATIC, 7, 81, 60, 8
     EDITTEXT IDC_MODIFIED_LABEL, 67, 81, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "存取日期：", IDC_STATIC, 7, 93, 60, 8
+    LTEXT "存取日期：(&A)", IDC_STATIC, 7, 93, 60, 8
```

Comment:
As far as I can see elsewhere in this file and others, in chinese/japanese/korean, the accelerator would precede the double-points
(so here for example it would need to be:
```suggestion
    LTEXT "存取日期(&A)：", IDC_STATIC, 7, 93, 60, 8
```
)

Reply:
Good catch, and easy to miss that on my host where the chinese ':' are only tofu-blocks. I actually am aware of the desired order. Resolved.
![image](https://github.com/reactos/reactos/assets/33393466/83abe669-490d-448b-9884-04242c1b74be)